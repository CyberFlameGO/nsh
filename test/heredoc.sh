foo=1234
cat << EOF > /dev/stderr
hello $foo
world
EOF
echo after heredoc > /dev/stderr
