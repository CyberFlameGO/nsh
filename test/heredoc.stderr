hello 1234
world
after heredoc
